spec/features/user/password_edit_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe "As a registered user,", type: :feature do
+  describe "when I visit my profile page, click on edit password link, fill out the new password fields, and click submit," do 
+    it "then I am returned to my profile page and I see a flash message telling me that my password is updated." do
+      user = User.create!(name: "Steve",
+                          address:"123 Main St.",
+                          city: "Fort Collins",
+                          state: "GA",
+                          zip: "66666",
+                          email: "chunky_lover@example.com",
+                          password: "123password")
+
+      # allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```

Comment:
that was weird that this wasn't working I'd be curious to see how you would tear down a stub like this one

Reply:
Maybe the test would have to be in another it do block? I am curious too.